src/style.css

```diff
@@ -0,0 +1,43 @@
+
+html {
+  box-sizing: border-box;
+}
+
+* {
+  margin: initial;
+}
+
+.stars__box {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
```suggestion
```
not need to set this styles. div has display block and will be render in column by default